components/virtual-node/virtual-node-rest-service-impl/build.gradle

```
@@ -27,6 +27,7 @@ dependencies {
     implementation project(':libs:lifecycle:lifecycle')
     implementation project(':libs:messaging:messaging')
     implementation project(':libs:membership:membership-common')
+    implementation project(':libs:rest:json-serialization')
```

Out of interest, why do we need this additional module dependency?

---

The tests `RequestFactoryImplTest` and `VirtualNodeValidationServiceImplTest` both need `import net.corda.rest.json.serialization.JsonObjectAsString` to get a JsonObject from a string

---

Ah, in this case it would more appropriate: 
```suggestion
    testImplementation project(':libs:rest:json-serialization')
```
and move it to the bottom of the file.

---

Makes sense to me. Done